Inferences/ALASCA/VariableElimination.cpp

```diff
@@ -207,13 +206,6 @@ ClauseIterator VariableElimination::applyRule(Clause* premise, FoundVariable<Num
   auto& K = found.eq;
   auto Ksize = K.size();
 
-  if (Ksize > 0) {
```

Comment:
Are we happy to sack the more detailed stuff?

I assume we newly only track existence of the new `InferenceRule::ALASCA_VARIABLE_ELIMINATION` clause(s)?

Reply:
I asked @joe-hauns how to categorize these (and all other arithmetic related extra stats), and he said they are not used currently and can be removed.